src/SFA.DAS.AdminService.Web/Infrastructure/RoatpTokenService.cs

```diff
@@ -1,35 +1,30 @@
 ﻿using Microsoft.Extensions.Hosting;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
 using SFA.DAS.AdminService.Settings;
+using Azure.Identity;
+using Azure.Core;
 
 namespace SFA.DAS.AdminService.Web.Infrastructure
 {
     public class RoatpTokenService : IRoatpTokenService
     {
         private readonly IWebConfiguration _configuration;
-        private readonly IHostingEnvironment _hostingEnvironment;
-        public RoatpTokenService(IWebConfiguration configuration, IHostingEnvironment hostingEnvironment)
+        private readonly IHostEnvironment _hostEnvironment;
+
+        public RoatpTokenService(IWebConfiguration configuration, IHostEnvironment hostEnvironment)
         {
             _configuration = configuration;
-            _hostingEnvironment = hostingEnvironment;
+            _hostEnvironment = hostEnvironment;
         }
 
         public string GetToken()
         {
-            if (_hostingEnvironment.IsDevelopment())
+            if (_hostEnvironment.IsDevelopment())
                 return string.Empty;
 
-            var tenantId = _configuration.RoatpApiAuthentication.TenantId;
-            var clientId = _configuration.RoatpApiAuthentication.ClientId;
-            var appKey = _configuration.RoatpApiAuthentication.ClientSecret;
-            var resourceId = _configuration.RoatpApiAuthentication.ResourceId;
-
-            var authority = $"https://login.microsoftonline.com/{tenantId}";
-            var clientCredential = new ClientCredential(clientId, appKey);
-            var context = new AuthenticationContext(authority, true);
-            var result = context.AcquireTokenAsync(resourceId, clientCredential).Result;
+            var result = new DefaultAzureCredential().GetTokenAsync(
```

Comment:
@ChrisJWoodcock if not using async in `GetTokenAsync()` is it worth using `GetToken()` instead?